committee.html

```diff
@@ -21,7 +21,19 @@ <h1>Committee</h1>
 <p>
   You can also find all of the agendas from our committee meetings <a href="../agendas">here</a>.
 <p>
-  Currently 13 elected members for 2021/22:
+  {% for committees in site.data.committee %}
+    {% assign committee = committees[1] %}
+    {% if committee.current %}
+      {% assign current_committee = committee %}
+      {% assign count = current_committee.people.size %}
+      {% for member in current_committee.people %}
+        {% if member.end-date %}
+          {% assign count = count | minus:1 %}
+        {% endif %}
+      {% endfor %}
+    {% endif %}
+  {% endfor %}
+  Currently {{ count }} elected members for 2021/22:
```

Comment:
It might also be worth dynamically updating the year here as well

Reply:
You're right. Does it matter whether the date format remains the same ?
For example, does it have to remain 2021/22 or will 2021/2022 be accepted ?
I'm asking because the latter is easier to implement as it's already stored in the YAML file.

Reply:
I don't think anyone would mind either way :) 
If anyone wants to change the formatting then they can just change it in the configuration file instead